src/views/components/display-html/DisplayHTML.native.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import Html from 'react-native-render-html';
+import { string } from 'prop-types';
+import { Box } from '../../components';
+
+class DisplayHTML extends Component {
+  static defaultProps = {
+    content: '',
+  }
+
+  static propTypes = { 
+    content: string,
+  };
+
+  render() { 
+    const { content } = this.props;
```

Comment:
Are we also able to refactor this prop (`content`) to be `html`? So it's super concise about what you have to pass in.